toptal/picasso
# chore(autocomplete): rename snapshot to match the example name

follow-up for #799

### Description

The example was renamed, but the image snapshot was not so running visual tests on the master branch with the update flag causes this unintentional change to appear.


### How to test

- See visual tests green
- Update visual tests without any local change

### Review
